### PR TITLE
feat(group): many-to-many relation between groups and users

### LINKED ISSUE #87: Legg til brukere på Group-modellen som en mange til mange relasjon gjennom Membership
Legg til en many-to-many referanse fra Group til User-modellen gjennom Membership slik vi har for Event og User gjennom `registered_users_list`. Dette gjør det enklere å ta i bruk denne relasjonen ved senere spørringer

### PR DESCRIPTION
**Pull request by Omjoa001**
## Proposed changes
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

Issue number: closes #87 

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] CHANGELOG.md has been updated. [Guide](https://tihlde.slab.com/posts/changelog-z8hybjom)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] The fixtures have been updated if needed (for migrations)
- [ x ] Pull request title follows [conventional commits](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) (`type(scope): description`)
- [ x ] Format and lint (`make format` and `make flake8`) was run locally without failures

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: app/group/models/group.py [hunk 1]
```diff
@@ -21,6 +21,14 @@ class Group(OptionalImage, BaseModel, BasePermissionModel):
     fine_info = models.TextField(default="", blank=True)
     type = EnumChoiceField(GroupType, default=GroupType.OTHER)
     fines_activated = models.BooleanField(default=False)
+    members = models.ManyToManyField(
+        User,
+        through="Membership",
+        through_fields=("group", "user"),
+        blank=True,
+        default=None,
+        verbose_name="group members",
```
Comment (olros) [inline app/group/models/group.py:30]: Stor G?
```suggestion
        verbose_name="Group members",
```

**Review by olros — APPROVED** [review]
Bør vi ha tilsvarende for membership histories også?

**Review by olros — CHANGES_REQUESTED** [review]
Slett begge de 2 migrasjonene og kjør `make migrations` igjen

**Comment by Omjoa001** [comment]
> Slett begge de 2 migrasjonene og kjør `make migrations` igjen

done

**Review by olros — APPROVED** [review]

**Pull request merged**